src/toil/common.py

```diff
@@ -213,6 +215,7 @@ def _addOptions(addGroupFn, config):
     addOptionFn("--maxParasolBatches", dest="maxParasolBatches", default=None,
                 help="Maximum number of batches Parasol is allowed to create - a batch \
                 is created for each job that has a unique set of resource requirements. Default=%i" % config.maxParasolBatches)
+    addOptionFn("--setEnv", dest="setEnv", default={}, help="Set of arbitrary environment variables for workers. Default=%i" % config.setEnv)
 
```

Comment:
Did you actually try this? How do I specify multiple variables on the same script invocation?